code/index.html

```diff
@@ -3,28 +3,110 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Rainy Night Publishing</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500&display=swap" rel="stylesheet">
+    
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header>
+    <img src="https://images.unsplash.com/photo-1515125520141-3e3b67bc0a88?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2102&q=80" alt="books in a shelf">
+    <div class="headline-text">
+      <h1>Rainy Nights Publishing</h1>
+      <h2>Where books come to life</h2>
+      <button type="button" class="header-button"><a href="#form-link">Get started</a></button>
+    </div>
+  </header>
+
+
+
 
     <!-- Signup form -->
+    <section class="form-wrapper" id="form-link">
+      <div class="form-header">
+        <h3>Publish with us</h3>
+        <p class="p-padding-none">* Required</p>
+      </div>
+    <form action="https://httpbin.org/anything" method="POST">
+      <div class="text-field">
+        <label for="first-name">First Name *</label>
+        <input type="text" id="first-name" name="first-name" required>
+        <br>
+        <label for="last-name">Last Name *</label>
+        <input type="text" id="last-name" name="last-name" required>
+        <br>
+        <label for="email">E-mail *</label>
+        <input type="email" id="email" name="email" required>
+      </div>
+        <br>
+        <br>
+        <label for="published">I have been published before</label>
+        <br>
+        <select id="published" name="published-before" value="yes">
+        <option value="Yes">Yes</option>
+        <br>
+        <option value="no">No</option>
+        </select>
+        <br>
+        <br>
+        <label for="write">I write:</label>
+        <br>
+        <input type="checkbox" value="romance" id="write" name="write">
+        <label for="romance">Romance</label>
+        <br>
+        <input type="checkbox" value="crime" id="write" name="write">
+        <label for="crime">Crime</label>
+        <br>
+        <input type="checkbox" value="non-fiction" id="write" name="write">
+        <label for="non-fiction">Non fiction</label>
+        <br>
+        <input type="checkbox" value="young-adults" id="write" name="write">
+        <label for="young-adults">Young adults</label>
+        <br>
+        <input type="checkbox" value="other" id="write" name="write">
+        <label for="other">Other</label>
+        <br>
+        <div class="button-wrapper">
+        <input type="submit" value="Submit" class="button">
+      </div>
+    </form>
+  </section>
+
```

Comment:
Your indentation is something else! It is so nice to read! And all the inputs' type and names are very well arranged.